### DISCUSSION TITLE
Select field not show init value on the DB

### DISCUSSION THREAD

**Discussion by lironsam** [filamentphp/filament#1232] [Help]
Hi,

I have a table with an attribute called "difficulty". It's basically just a varchar field which can accept three possibilities: easy, medium and difficult.

On Filament Admin, I would like to add to my ressource form, the possibility to change the difficulty.

So far, so good, I'm doing this:

```
Forms\Components\Select::make('difficulty')
                ->label('Difficulty')
                ->options($options = ["easy"=>"Easy", "medium"=>"Medium", "Difficult"=>"Difficult"]),
```

However, when I edit my ressource that has already an attribute initialized with "medium", the form does not show it:
<img width="1075" alt="image" src="https://user-images.githubusercontent.com/8614763/149374532-cd7578b1-f1a3-40ee-baaa-fde4577c3ca9.png">

Therefore, I need to select manually each time, otherwise, the field will be updated with an empty value.

Did I miss something to show the value ?

I tried with default and some callback, but it did not work.

thank a lot !

**Comment by danharrin** [comment]
Hey, this should "just work", so I think something is probably wrong in your setup. Double check the casing of the values etc, and that the `<option>s` in HTML are correct.
